From 55583787ce9ed880c7379ec095c5a98521653a17 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Wed, 30 Jun 2021 23:27:04 +0200
Subject: [PATCH 3/5] kbuild: add proper bool for MODULE_COMPRESS_ZSTD_ULTRA

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 init/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/Kconfig b/init/Kconfig
index 558bc5f93..e07c47d1d 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -2268,7 +2268,7 @@ endchoice
 
 config MODULE_COMPRESS_ZSTD_ULTRA
 	depends on MODULE_COMPRESS_ZSTD
-	bool "ZSTD"
+	bool "Enable ZSTD ultra compression"
 	help
 	  Compress modules with ZSTD using the highest possible compression.
 
-- 
2.33.0.rc0.52.ge5a14ddd2d

